lib/open_rtb_ecto/v2/bid_request/video.ex

```diff
@@ -37,6 +37,7 @@ defmodule OpenRtbEcto.V2.BidRequest.Video do
     field(:mincpmpersec, :float)
     field(:slotinpod, :integer)
     field(:placement, :integer)
+    field(:plcmt, :integer)
```

Comment:
what version adds this field? I dont see it in 2.5 or 2.6

Reply:
ah i see here: https://iabtechlab.com/blog/march-2023-update-to-openrtb-is-now-ready-for-implementation/

Reply:
https://iabtechlab.com/blog/updates-to-openrtb-for-public-comment/
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo

Reply:
https://iabtechlab.com/blog/march-2023-update-to-openrtb-is-now-ready-for-implementation/